### ISSUE TITLE
[Feature request] Add to Microsoft Store and environmental variables, like Sysinternals Suite

### ISSUE THREAD

**Issue by o-l-a-v** [microsoft/Microsoft-Win32-Content-Prep-Tool#66]
Add this tool to Microsoft Store, just like Microsoft have done with the Sysinternals Suite:

* https://www.microsoft.com/en-us/store/apps/9p7knl5rwt25

And add it to environmental variables / PATH at the same time.

Other honerable mentions:

* [Rufus](https://www.microsoft.com/en-us/store/apps/9pc3h3v7q9ch)

**Comment by jbolduan** [comment]
I agree with this, it would also be handy to have it in winget to help keeping the tool up to date on a packaging device.

**Comment by o-l-a-v** [comment]
With Winget now supporting portable apps, maybe that's a better option than Microsoft Store.
